microservices/cart/src/aggregates.py

```diff
@@ -0,0 +1,36 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from minos.common import (
+    Aggregate,
+    AggregateRef,
+    Entity,
+    EntitySet,
+    ModelRef,
+)
+
+
+class Product(AggregateRef):
+    """Product AggregateRef class."""
+
+    title: str
+    description: str
+    price: float
+
+
+class CartEntry(Entity):
+    """Cart Item DeclarativeModel class."""
+
+    quantity: int
+    product: ModelRef[Product]
+
+
+class Cart(Aggregate):
+    """Cart Aggregate class."""
+
+    user: int
+    entries: EntitySet[CartEntry]
```

Comment:
Could you reorder classes to follow the `Cart`, `CartEntry`, `Product` order? (You might need to import `from __future__ import annotations`

Reply:
Done!